v1.0/backup.md

```diff
@@ -89,7 +89,7 @@ Only the `root` user can run `BACKUP`.
 | `table_pattern` | The table or [view](views.html) you want to back up. |
 | `name` | The name of the database you want to back up (i.e., create backups of all tables and views in the database).|
 | `destination` | The URL where you want to store the backup.<br/><br/>For information about this URL structure, see [Backup File URLs](#backup-file-urls). |
-| `AS OF SYSTEM TIME timestamp` | Back up data as it existed as of [`timestamp`](timestamp.html). However, the `timestamp` must be more recent than your cluster's last garbage collection (which defaults to occur every 24 hours, but is [configurable per table](configure-replication-zones.html#replication-zone-format)). |
+| `AS OF SYSTEM TIME timestamp` | Back up data as it existed as of `timestamp`. {% include aost-v1.0.md %} However, the `timestamp` must be more recent than your cluster's last garbage collection (which defaults to occur every 24 hours, but is [configurable per table](configure-replication-zones.html#replication-zone-format)). |
```

Comment:
Not sure exactly why, but includes generally (or always?) don't work in basic markdown table cells:

![screen shot 2018-04-24 at 10 01 19 am](https://user-images.githubusercontent.com/15893490/39192161-98882da2-47a6-11e8-939d-e8fb37d11f0b.png)

Unless you can figure out how to get that to work, we have a few other options:

1. Just duplicate the content wherever it needs to be.
2. Create a separate section for `AS OF SYSTEM TIME`, link from the cell to that section, and then use that include in the new section. We do that for the `--locality` flag on `start-a-node.md`, for example. Not sure if there's enough content here to warrant this approach. Your call.